css/_base.scss

```diff
@@ -162,6 +162,15 @@ input, button, select, textarea {
   cursor: move;
 }
 
+.favourite-button {
```

Comment:
I don't like this here in the `base.scss`.... Should we create a `c-favourite-button`? What do you think?

Reply:
makes sense, will move it out to its own file !

Reply:
its a bit messy, we are shadow applying styles on all places where the favorite button is, so still needed to update lots of files, but well... its working a bit better now